src/transformStateWithClones.js

```diff
@@ -65,7 +65,55 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const operationHandlers = initOperationHandlers();
+  const objectStatesHistory = [];
+  let currentState = { ...state };
+
+  for (const { operation, properties } of transforms) {
+    const newObjectState
+    = operationHandlers[operation](currentState, properties);
```

Comment:
```suggestion
    const newObjectState = operationHandlers[operation](
      currentState, properties,
    );
```
ми так не переносимо код зазвичай